osbs/api.py

```diff
@@ -505,7 +505,8 @@ def _do_create_prod_build(self, git_uri, git_ref,
             info_url_format=self.build_conf.get_info_url_format(),
             artifacts_allowed_domains=self.build_conf.get_artifacts_allowed_domains(),
             low_priority_node_selector=self.build_conf.get_low_priority_node_selector(),
-            equal_labels=self.build_conf.get_equal_labels()
+            equal_labels=self.build_conf.get_equal_labels(),
+            node_selector_platform=self.build_conf.get_node_selector_platform(platforms)
```

Comment:
This should be `(platform)`, not `(platforms)`. Also can we call the parameter `platform_node_selector`, for comparison with `low_priority_node_selector` above it.